cmd/gardener-operator/app/app.go

```diff
@@ -180,7 +180,7 @@ func run(ctx context.Context, cancel context.CancelFunc, log logr.Logger, cfg *o
 
 	log.Info("Adding webhook handlers to manager")
 	if err := webhook.AddToManager(mgr); err != nil {
-		return fmt.Errorf("failed adding webhook handlers to manager: %w", err)
+		return err
```

Comment:
Why this was changed?

Reply:
The message is already specified in `pkg/operator/webhook/add.go`.